coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -60,12 +60,31 @@
                       required></v-text-field>
                   </div>
 
-                  <BaseAddress ref="baseAddressNew"
-                    :address="director.deliveryAddress"
-                    :editing="true"
-                    :schema="addressSchema"
-                    @update:address="baseAddressWatcher"
-                  />
+                  <label class="address-sub-header">Delivery Address</label>
+                  <div class="address-wrapper">
+                    <BaseAddress ref="baseAddressNew"
+                       v-bind:editing="true"
+                       :schema="addressSchema"
+                       @update:address="baseAddressWatcher"
+                    />
+                  </div>
+                  <div class="form__row" v-if="entityFilter(EntityTypes.BCorp)">
+                    <v-checkbox
+                      class="inherit-checkbox"
+                      label="Mailing Address same as Delivery Address"
+                      v-model="inheritDeliveryAddress"
+                    ></v-checkbox>
+                    <div v-if="entityFilter(EntityTypes.BCorp) && !inheritDeliveryAddress">
```

Comment:
You don't need to check "if BCorp" because you're already in a div where you checked this.

Reply:
Good catch, definite oops i left in during my testing. Thank you!